valkeycompat/command.go

```diff
@@ -529,6 +529,7 @@ func (cmd *SliceCmd) from(res valkey.ValkeyResult) {
 		}
 	}
 	cmd.SetVal(vals)
+	cmd.SetErr(err)
```

Comment:
Hi @zeeshan-zomato, thanks! But I wonder if we should fix this in the Pipeline.Exec instead.

Reply:
And TxPipeline.Exec. Just use `rets[i].SetErr(nil)` to clear all responses before `rets[i].from(...)`